style-guide.md

```diff
@@ -0,0 +1,189 @@
+# Cincytech slack community Job Description Style Guide
+
+Finding the right person for a job is difficult. And writing a job descriptions
+to attract that person is hard. Provide too much detail and no one will ever
+read it. Provide too little detail and no one would dare apply. It's an art
+form, and attempting to strike a balance between too little and too much can be
+time consuming and the results can be unpredictable.
+
+
+This document is an attempt to summarize what we, the cincytech slack community
+feel is a good job description for jobs in software development. It is not meant
+to be exhaustive or authoritative, instead it is merely a set of principles to
+keep in mind when you are writing the job description to find the perfect person you need
+for the job.
```

Comment:
Unfortunately, I'm not _entirely_ sure. Certainly the parts about the company/project are quite important as are details of the working environment. In *my* head, I think about the typical questions about a company/project/*job* and whether the answer (or partial answer) can be found in the description. To me, it seems like most "job ads" spend almost all their energy defining the ideal candidate and far too little in defining the environment in which that candidate will find themselves working.

_(Don't know if that's helping or not.)_